src/ontology/mondo-edit.obo

```diff
@@ -143956,7 +143961,7 @@ property_value: seeAlso https://rarediseases.info.nih.gov/diseases/5887/bannayan
 
 [Term]
 id: MONDO:0007925
-name: chromosome 5q deletion syndrome
+name: Myelodysplastic syndrome associated with isolated del(5q)
```

Comment:
the label should be lowercase unless it is a proper name

Reply:
done